prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -645,7 +646,10 @@ class ActionHistory {
                     msg = "reportFiles is null"
                 } else {
                     if (tasks.size != reportFiles.size) {
-                        msg = "Different result count: Got ${tasks.size} headers but ${reportFiles.size} reportFiles"
+                        msg = "Different report_file count: Got ${tasks.size} TASKS," +
```

Comment:
A report that is missing from the REPORT_FILE table is an unrecoverable bug.   However currently such bugs cause an entire batch to fail, even if one report is missing.   To prevent this badness, I've changed to code to skip around these errors, but this at least helps to log the details of what occurred.